prody/proteins/ciffile.py

```diff
@@ -290,6 +290,8 @@ def _parseMMCIFLines(atomgroup, lines, model, chain, subset,
         startswith = line.split()[fields['group_PDB']]
 
         atomname = line.split()[fields['auth_atom_id']]
+        if atomname.find('"') != -1:
+            atomname = atomname[1:4]
```

Comment:
The fix looks fine to me. But Since any quotes in the atom name will trigger this, maybe the fix can be more aggressive too?
```
atomname = atomname.replace('"', '')
```

Reply:
> The fix looks good and should make it to the release. However, I wonder how the quotes get into the atom's name in the first place.

They're in the mmCIF file:
```
In [3]: fi = open('/mnt/c/Users/james/OneDrive/Documents/MC_fellowship/4_other_systems/1_CCT/4v8r.cif', 'r')

In [4]: lines = fi.readlines()

In [5]: fi.close()

In [6]: line = lines[137924]

In [7]: line
Out[7]: 'HETATM 128757 O  "O5\'" . ADP VD 9  .   ? 182.720 -58.776 -22.315 1.00 146.40 ? 4601 ADP Bz "O5\'" 1 \n'

In [8]: line.split()
Out[8]: 
['HETATM',
 '128757',
 'O',
 '"O5\'"',
 '.',
 'ADP',
 'VD',
 '9',
 '.',
 '?',
 '182.720',
 '-58.776',
 '-22.315',
 '1.00',
 '146.40',
 '?',
 '4601',
 'ADP',
 'Bz',
 '"O5\'"',
 '1']

In [9]: line.split()[3]
Out[9]: '"O5\'"'
```

Reply:
Maybe I should be left aggressive actually instead. It could be that there are some we want.

Reply:
How about this?
```
        if atomname.startswith('"') and atomname.endswith('"'):
            atomname = atomname[1:-1]
```